ehr_billing/resources/schemas/dbscripts/sqlserver/ehr_billing-20.000-20.001.sql

```diff
@@ -0,0 +1,43 @@
+EXEC core.fn_dropifexists 'invoicedItems', 'ehr_billing', 'INDEX', 'EHR_BILLING_INVOICEDITEMS_INVNUM_INDEX';
```

Comment:
@labkey-jeckels 
Included this sql script to increase the size of invoiceNumber column as they were getting huge on repeat refunds of the ITs and the rows were not getting inserted. AbstractQueryService did not throw any error for this situation and the insert rows were just ignored.